bazel/WORKSPACE

```diff
@@ -61,11 +61,11 @@ tf_workspace(
 http_archive(
     name = "ngraph",
     build_file = "//:bazel/ngraph.BUILD",
-    sha256 = "24e723f7ed47e2c0068fb5868d8ca88542948a13c9de063d8108f2b67785b089",
-    strip_prefix = "ngraph-0.28.0-rc.1",
+    sha256 = "",
+    strip_prefix = "ngraph-8e37fbfaeb7ee827746d607d9688dc509a25fe77",
     urls = [
-        "https://mirror.bazel.build/github.com/NervanaSystems/ngraph/archive/v0.28.0-rc.1.tar.gz",
-        "https://github.com/NervanaSystems/ngraph/archive/v0.28.0-rc.1.tar.gz",
+        "https://mirror.bazel.build/github.com/NervanaSystems/ngraph/archive/8e37fbfaeb7ee827746d607d9688dc509a25fe77.tar.gz",
```

Comment:
You can use `https://github.com/NervanaSystems/ngraph/archive/tf-bridge-ie-backend.tar.gz` at both places here.